lib/sles4sap/qesap/qesap_utils.pm

```diff
@@ -0,0 +1,84 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Internal utility functions only used by qesapdeployment
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+## no critic (RequireFilenameMatchesPackage);
+
+=encoding utf8
+
+=head1 NAME
+
+    qesap-utils lib
+
+=head1 COPYRIGHT
+
+    Copyright 2025 SUSE LLC
+    SPDX-License-Identifier: FSFAP
+
+=head1 AUTHORS
+
+    QE SAP <qe-sap@suse.de>
+
+=cut
+
+package sles4sap::qesap::qesap_utils;
+
+use strict;
+use warnings;
+use Carp qw(croak);
+use Mojo::JSON qw(decode_json);
+use Exporter 'import';
+use testapi;
+
+our @EXPORT = qw(
+  qesap_is_job_finished
+);
+
+=head1 DESCRIPTION
+
+    Package with utils internal methods
```

Comment:
s/Package with utils internal methods/Package with util qesap-deployment functions/

Methods means it's OO, which this isn't.